spec/std/http/http_spec.cr

```diff
@@ -3,24 +3,24 @@ require "http"
 
 describe HTTP do
   it "parses RFC 1123" do
-    time = Time.new(1994, 11, 6, 8, 49, 37)
+    time = Time.utc(1994, 11, 6, 8, 49, 37)
     HTTP.parse_time("Sun, 06 Nov 1994 08:49:37 GMT").should eq(time)
   end
 
   it "parses RFC 1123 without day name" do
-    time = Time.new(1994, 11, 6, 8, 49, 37)
+    time = Time.utc(1994, 11, 6, 8, 49, 37)
```

Comment:
Should not be UTC (following line check it's a GMT time)

Reply:
No, this is completely valid.
* HTTP dates represented in the RFC 1123 format are usually marked as `GMT` but are really UTC times, because the standard doesn't include `UTC`. This is also how the Crystal implementation works.
* This method is used to *create* a time instance refering to a specific point in time. For the equality check, the time zone does not matter, because the values are compared based on UTC anyway.
* Apart from that, there is not really a difference between dates in GMT and UTC zones at all in Crystal.

Reply:
My bad, forgot GMT was a timezone.